docker-compose.gh.yml

```diff
@@ -0,0 +1,111 @@
+services:
+    reverse-proxy:
+        image: traefik:v2.10
+        command: --api.dashboard=true --api.insecure=true --providers.docker --entrypoints.web.address=:80
+        ports:
+            - "80:80"
+            - "8080:8080"
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock
+        networks:
+            - agenta-network
+
+    agenta-backend:
+        container_name: agenta-backend-1
+        image: ghcr.io/agentaai/agenta-backend
+        environment:
+            - MONGODB_URI=mongodb://username:password@mongo:27017
+            - ENVIRONMENT=development
+            - BARE_DOMAIN_NAME=localhost
+            - DOMAIN_NAME=http://localhost
+            - FEATURE_FLAG=oss
+        command:
+          [
+              "uvicorn",
+              "agenta_backend.main:app",
+              "--host",
+              "0.0.0.0",
+              "--port",
+              "8000",
+              "--reload",
+              "--log-level",
+              "info",
+              "--root-path",
+              "/api"
+          ]
+        volumes:
+          - ./agenta-backend/agenta_backend:/app/agenta_backend
```

Comment:
I thought you removed the volumes? 

The goal is to be able run agenta from the dockerfile only, without even needing to clone the repository.

Reply:
Oh, yes. I added them to test the image pulled from my container registry.